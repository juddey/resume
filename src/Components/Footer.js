import React from 'react'
import Typography from '@material-ui/core/Typography'
// import styles from './Styles/EducationStyle'

const Footer = () => (
  <Typography style={{ color: 'black' }}>
    Reskilled in web and mobile development from a ERP software implementation
    career, I'm looking for remote-based short to mid term contracts. Rates
    negotiable.
  </Typography>
)
export default Footer
