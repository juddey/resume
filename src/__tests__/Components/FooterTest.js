import React from 'react'
import {
  createComponentWithIntl,
  shallowWithIntl
} from '../../../src/Support/Utils'
import Footer from '../../Components/Footer'

const div = document.createElement('div')
const wrapper = shallowWithIntl(<Footer />)

test('component exists', () => {
  expect(wrapper.length).toBe(1)
})

test('component structure', () => {
  expect(wrapper.name()).toEqual('WithStyles(Typography)')
  expect(wrapper.children().length).toEqual(1)
})

test('renders to snapshot', () => {
  const tree = createComponentWithIntl(<Footer />, div).toJSON()
  expect(tree).toMatchSnapshot()
})
