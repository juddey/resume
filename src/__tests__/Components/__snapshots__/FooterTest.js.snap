// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to snapshot 1`] = `
<div
  className="MuiPaper-root-1 MuiPaper-elevation4-7 MuiPaper-rounded-2"
  style={
    Object {
      "backgroundColor": "#AAAAAA",
    }
  }
>
  <p
    className="MuiTypography-root-28 MuiTypography-body1-37"
  >
    Reskilled in web and mobile development from a ERP software implementation career, I'm looking for remote-based short to mid term contracts. Rates negotiable.
  </p>
</div>
`;
